bootloader/grub.go

```diff
@@ -35,8 +39,15 @@ var (
 	_ installableBootloader             = (*grub)(nil)
 	_ RecoveryAwareBootloader           = (*grub)(nil)
 	_ ExtractedRunKernelImageBootloader = (*grub)(nil)
+	r *rand.Rand
 )
 
+func init() {
+	// seed random number generator for the filenames to use for intermediate
+	// kernel.efi symlinks
+	r = rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
What does this protect against? I'm worried that this will not be reliable in devices that lack RTC. Can you add a comment into the code as to what's the consequence of poor randomization?